pkg/dsync/drwmutex.go

```diff
@@ -292,27 +315,19 @@ func lock(ctx context.Context, ds *Dsync, locks *[]string, id, source string, is
 
 	wg.Wait()
 
-	return quorum
+	return quorumMet
 }
 
-// quorumMet determines whether we have acquired the required quorum of underlying locks or not
-func quorumMet(locks *[]string, isReadLock bool, quorum, quorumReads int) bool {
-
+// checkQuorumMet determines whether we have acquired the required quorum of underlying locks or not
+func checkQuorumMet(locks *[]string, isReadLock bool, quorum int) bool {
```

Comment:
we could remove `isReadLock` parameter now that `quorum` is computed accordingly by the caller.

Reply:
^^ true, isReadLock not being used anymore